templates/product/_items.html

```diff
@@ -21,7 +21,7 @@
         <div class="panel-footer">
           {% if availability.available %}
             {% price_range availability.price_range %}
-            {% if availability.discount %}
+            {% if availability.discount != None %}
```

Comment:
Such checks in templates always feel a little bit fishy to me and I was wondering if we shouldn't change the `availability.discount` to always return `TaxedMoney`. But it actually makes sense (e.g. in API) to return `None` if there is no discount. 
Anyway, there is also the `availability.on_sale` flag which seems to be meant exactly for that use-case. Could you test if we could use it here?

Reply:
Indeed. Thansk for the info. ```on_sale``` is perfect for this case. I've updated the PR.